### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by lazd**
## Description

Fix icon usage information #63

Add info on custom properties usage #133

## Types of changes

- [Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +282,12 @@ Check out the [contributing guidelines](.github/CONTRIBUTING.md)!
 
 [topdoc-link]: https://github.com/Topdoc/topdoc/wiki
 
+### Updating internal dependencies
+
+*Internal-to-Adobe only - for now*
+
+Locally clone the `spectrum-css-deps` repo. This repo contains internal, non open source dependencies needed by Spectrum CSS. Keep `spectrum-css-deps` as a sibling repo to `spectrum-css`. In `spectrum-css-deps`, run `npm run build` to generate the dependencies and `npm run copy` to copy the generated dependencies into `spectrum-css`. Then go ahead and review and commit any changes.
+
```
Comment (lazd) [inline README.md]: Also moved this below near releasing, since that makes more sense in context.

File: README.md [hunk 2]
```diff
@@ -162,47 +199,6 @@ loadIcons('icons/spectrum-icons.svg');
 
 You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](http://opensource.adobe.com/spectrum-css/icons/) and click on any icon to get the markup.
 
-### Swapping out icons sets for scaling
```
Comment (lazd) [inline README.md]: This just isn't relevant anymore, since we advise people to use `spectrum-css-icons.svg`, which has both.

File: README.md [hunk 3]
```diff
@@ -128,7 +165,7 @@ Note that the Spectrum CSS UI icons must change as well, see below for a full ex
 
 ### Importing Icons
 
-To get icons, you'll need to use the `loadIcons()` function. This function lives in the `icons/loadIcons.js` file and has the following signature:
+To get icons, you'll need to use the `loadIcons()` function. This function lives in the [`loadicons` project](https://www.npmjs.com/package/loadicons) as well as `icons/loadIcons.js` and has the following signature:
```
Comment (lazd) [inline README.md]: Finally referencing the open source project, yay.

File: README.md [hunk 4]
```diff
@@ -2,40 +2,77 @@
...
+<!-- Include only the scales and colorstops your application needs, be sure to use the *-unique.css files -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-medium-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-large-unique.css">
+
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-light-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-dark-unique.css">
+
+<!-- Include index-vars.css for all components you need -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/components/icon/index-vars.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.10.0/dist/components/button/index-vars.css">
```
Comment (lazd) [inline README.md]: Had reservations about including `unpkg` in here, but honestly it makes the usage really obvious and it's copy-paste ready, so I think it's good. Thoughts?

File: README.md [hunk 5]
```diff
@@ -2,40 +2,77 @@
...
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-medium-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-large-unique.css">
+
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-light-unique.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/vars/spectrum-dark-unique.css">
+
+<!-- Include index-vars.css for all components you need -->
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/components/icon/index-vars.css">
+<link rel="stylesheet" href="https://unpkg.com/@adobe/spectrum-css@2.x/dist/components/button/index-vars.css">
+```
```
Comment (lazd) [inline README.md]: Had reservations about including unpkg in here, but honestly it makes the usage really obvious and it's copy-paste ready, so I think it's good. Thoughts?

File: README.md [hunk 6]
```diff
@@ -2,40 +2,77 @@
...
-4. [Testing](#testing-with-the-visual-regression-tools)
-5. [Contributing](#contributing)
+    1. [Single-stop Strategy](#single-stop-strategy)
+1. [Building](#building)
+1. [Testing](#testing-with-the-visual-regression-tools)
+1. [Contributing](#contributing)
+
+## Show me a demo
+
+Check out [the documentation here](http://opensource.adobe.com/spectrum-css/), which shows every Spectrum CSS component and all of its variations on the same page.
```
Comment (lazd) [inline README.md]: I figured just maybe we should link to our docs here in the readme, ya know?  ;)

File: README.md (hunk 5 above)
Comment (GarthDB) [reply README.md]: I think it's great to explain how we use it.

File: README.md (hunk 3 above)
Comment (GarthDB) [reply README.md]: 👍

**Pull request merged**